charts/virtaitech-orion-vgpu/v1.0.0/values.yaml

```diff
@@ -0,0 +1,95 @@
+image:
+  pullPolicy: IfNotPresent
+imagePullSecrets: []
+
+resources:
+  # Interesting Config:
+  # resource name exposed by Orion
+  name: "virtaitech.com/gpu"
+
+monitor:
+  collectorPath: /run/prometheus
+  image:
+    exporter:
+      # nvidia/dcgm-exporter
+      repository: virtaitech/dcgm-export
+      version: "2.2"
+    prometheus:
+      # quay.io/prometheus/node-exporter
+      repository: virtaitech/node-exporter
+      version: "2.2"
+  # this port is also used by controller to scrap info.
+  # should not change this value
+  port: 9100
+
+controller:
+  version: "2.2"
+  name: "orion-controller"
+  port: "15500"
+  portName0: "controller"
+  portName1: "port1"
+  portName2: "webportal"
+  nodeport: 30007
+  service: orion-controller
+  image:
+    prom: virtaitech/prometheus
+    repository: virtaitech/orion-controller
+    version: "2.2"
+  # Interesting Config:
+  password: "admin"
```

Comment:
password better empty. Fill it out at creation time.

Reply:
Changed default password to blank.